### PR TITLE
♿ Adds Alt & aria-label to Placeholder Images

### PR DESCRIPTION
**Pull request by gulliverhan**
addresses Issue #5508

Adds alt tag & propagates aria label to the placeholder images for 

amp-brid-player
amp-jwplayer
amp-kaltura-player
amp-springboard-player
amp-youtube

### CODE REVIEW

File: extensions/amp-brid-player/0.1/amp-brid-player.js [hunk 1]
```diff
@@ -185,24 +185,33 @@ class AmpBridPlayer extends AMP.BaseElement {
...
-    		  'https://cdn.brid.tv/live/partners/' +
-    		  encodeURIComponent(partnerID) + '/snapshot/' +
-    		  encodeURIComponent(feedID) + '.jpg');
+          'https://cdn.brid.tv/live/partners/' +
+          encodeURIComponent(partnerID) + '/snapshot/' +
+          encodeURIComponent(feedID) + '.jpg');
       placeholder.setAttribute('layout', 'fill');
       placeholder.setAttribute('placeholder', '');
       placeholder.setAttribute('referrerpolicy', 'origin');
+      placeholder.setAttribute('alt', 'loading video');
```
Comment (cathyxz) [inline extensions/amp-brid-player/0.1/amp-brid-player.js]: This line is duplicated below under the `else` block. You can remove it.

**Comment by gulliverhan** [comment]
well spotted - have removed that line

**Comment by cathyxz** [comment]
@gulliverhan could you merge master onto this branch to incorporate the most recent test fixes? This will make CI more reliable and likely to pass. Sorry about that.

**Comment by cathyxz** [comment]
Thank you!

**Pull request merged**